PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -29,14 +29,25 @@ public void onClick(View v) {
 			}
 		});
 
-		ImageButton start = (ImageButton) findViewById(R.id.startButtonMain);
-		start.setOnClickListener(new View.OnClickListener() {
-			@Override
-			public void onClick(View v) {
-				Intent myIntent = new Intent(StartActivity.this, MapActivity.class);
-				startActivityForResult(myIntent, 0);
-			}
-		});
 
-	}
+        ImageButton start = (ImageButton) findViewById(R.id.startButtonMain);
+        start.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (!previouslyStarted) {
+                    startActivity(new Intent(StartActivity.this, AvatarRoomActivity.class));
+                } else {
+                    startActivity(new Intent(StartActivity.this, MapActivity.class));
+                }
+
+            }
+        });
+
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        previouslyStarted = prefs.getBoolean(getString(R.string.pref_previously_started), false);
+    }
```

Comment:
@alma-castant I have made the change, now the value is refreshed when this start activity opens again and it is fetched from shared preferences itself.